bindings/rust-bindings/src/chain_list.rs

```diff
@@ -0,0 +1,64 @@
+//! List of OP Stack chains.
+
+use alloc::{string::String, vec::Vec};
+
+/// List of Chains.
+#[derive(Debug, Clone, Default, Hash, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
+pub struct ChainList {
+    /// List of Chains.
+    pub chains: Vec<Chain>,
+}
+
+/// A Chain Definition.
+#[derive(Debug, Clone, Default, Hash, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
+pub struct Chain {
+    /// The name of the chain.
+    pub name: String,
+    /// Chain identifier.
+    pub identifier: String,
+    /// Chain ID.
+    pub chain_id: u64,
+    /// List of RPC Endpoints.
+    pub rpc: Vec<String>,
+    /// List of Explorer Endpoints.
+    pub explorers: Vec<String>,
+    /// The Superchain Level.
+    pub superchain_level: u64,
+    /// The Superchain Parent.
+    pub parent: SuperchainParent,
+}
+
+/// A Chain Parent
+#[derive(Debug, Clone, Default, Hash, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct SuperchainParent {
+    /// The parent type.
+    pub r#type: String,
+    /// The chain identifier.
+    pub chain: String,
+}
+
+impl SuperchainParent {
+    /// Returns the chain id for the parent.
+    pub fn chain_id(&self) -> u64 {
+        match self.chain.as_ref() {
+            "mainnet" => 1,
+            "sepolia" => 11155111,
+            "sepolia-dev-0" => 11155420,
```

Comment:
This is not the devnet; `11155420` is OP Sepolia (prod testnet)

Reply:
Good callout - took an AI to the knee